tests/main/desktop-portal-filechooser/task.yaml

```diff
@@ -53,6 +53,8 @@ execute: |
 
     echo "The confined application can write files via the portal"
     [ ! -f /tmp/file-to-write.txt ]
+    touch /tmp/file-to-write.txt
```

Comment:
I think we want a comment here why we create the file, probably fine to just copy most of what @bboozzoo wrote in https://github.com/snapcore/snapd/pull/6541#discussion_r260633830

Reply:
@mvo5 sure